2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Configure stats pid to 41656
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/.config/wandb/settings
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/settings
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_init.py:_log_setup():520] Logging user logs to /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/run-20240522_112048-ud4mi53w/logs/debug.log
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_init.py:_log_setup():521] Logging internal logs to /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/run-20240522_112048-ud4mi53w/logs/debug-internal.log
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_init.py:init():560] calling init triggers
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-05-22 11:20:48,242 INFO    MainThread:41656 [wandb_init.py:init():585] re-initializing run, found existing run on stack: uhh00i6z
2024-05-22 11:20:48,243 INFO    MainThread:41656 [wandb_run.py:_finish():2103] finishing run counterfactuals/sae_concept_eraser/uhh00i6z
2024-05-22 11:20:48,243 ERROR   MainThread:41656 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-05-22 11:20:48,243 INFO    MainThread:41656 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-22 11:20:48,243 INFO    MainThread:41656 [wandb_init.py:_jupyter_teardown():448] cleaning up jupyter logic
2024-05-22 11:20:48,243 INFO    MainThread:41656 [wandb_run.py:_atexit_cleanup():2343] got exitcode: 0
2024-05-22 11:20:48,244 INFO    MainThread:41656 [wandb_run.py:_restore():2326] restore
2024-05-22 11:20:48,244 INFO    MainThread:41656 [wandb_run.py:_restore():2332] restore done
2024-05-22 11:20:55,315 INFO    MainThread:41656 [wandb_run.py:_footer_history_summary_info():3994] rendering history
2024-05-22 11:20:55,315 INFO    MainThread:41656 [wandb_run.py:_footer_history_summary_info():4026] rendering summary
2024-05-22 11:20:55,327 INFO    MainThread:41656 [wandb_run.py:_footer_sync_info():3953] logging synced files
2024-05-22 11:20:55,520 INFO    MainThread:41656 [wandb_init.py:init():610] starting backend
2024-05-22 11:20:55,520 INFO    MainThread:41656 [wandb_init.py:init():614] setting up manager
2024-05-22 11:20:55,521 INFO    MainThread:41656 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-05-22 11:20:55,521 INFO    MainThread:41656 [wandb_init.py:init():622] backend started and connected
2024-05-22 11:20:55,524 INFO    MainThread:41656 [wandb_run.py:_label_probe_notebook():1328] probe notebook
2024-05-22 11:20:55,524 INFO    MainThread:41656 [wandb_run.py:_label_probe_notebook():1338] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-05-22 11:20:55,524 INFO    MainThread:41656 [wandb_init.py:init():711] updated telemetry
2024-05-22 11:20:55,540 INFO    MainThread:41656 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-05-22 11:20:57,015 INFO    MainThread:41656 [wandb_run.py:_on_init():2396] communicating current version
2024-05-22 11:20:57,098 INFO    MainThread:41656 [wandb_run.py:_on_init():2405] got version response 
2024-05-22 11:20:57,098 INFO    MainThread:41656 [wandb_init.py:init():795] starting run threads in backend
2024-05-22 11:21:13,762 INFO    MainThread:41656 [wandb_run.py:_console_start():2374] atexit reg
2024-05-22 11:21:13,763 INFO    MainThread:41656 [wandb_run.py:_redirect():2229] redirect: wrap_raw
2024-05-22 11:21:13,764 INFO    MainThread:41656 [wandb_run.py:_redirect():2294] Wrapping output streams.
2024-05-22 11:21:13,764 INFO    MainThread:41656 [wandb_run.py:_redirect():2319] Redirects installed.
2024-05-22 11:21:13,934 INFO    MainThread:41656 [wandb_init.py:init():838] run started, returning control to user process
2024-05-22 11:23:26,450 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-22 11:23:26,451 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Configure stats pid to 41656
2024-05-22 11:23:26,451 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/.config/wandb/settings
2024-05-22 11:23:26,451 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/settings
2024-05-22 11:23:26,451 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-22 11:23:26,451 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-22 11:23:26,451 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-22 11:23:26,451 INFO    MainThread:41656 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-22 11:23:26,451 INFO    MainThread:41656 [wandb_init.py:_log_setup():520] Logging user logs to /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/run-20240522_112326-umxxchse/logs/debug.log
2024-05-22 11:23:26,451 INFO    MainThread:41656 [wandb_init.py:_log_setup():521] Logging internal logs to /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/run-20240522_112326-umxxchse/logs/debug-internal.log
2024-05-22 11:23:26,451 INFO    MainThread:41656 [wandb_init.py:_jupyter_setup():466] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x16a436540>
2024-05-22 11:23:26,453 INFO    MainThread:41656 [wandb_init.py:init():560] calling init triggers
2024-05-22 11:23:26,453 INFO    MainThread:41656 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-05-22 11:23:26,453 INFO    MainThread:41656 [wandb_init.py:init():585] re-initializing run, found existing run on stack: ud4mi53w
2024-05-22 11:23:26,454 INFO    MainThread:41656 [wandb_run.py:_finish():2103] finishing run counterfactuals/sae_concept_eraser/ud4mi53w
2024-05-22 11:23:26,454 INFO    MainThread:41656 [wandb_run.py:_atexit_cleanup():2343] got exitcode: 0
2024-05-22 11:23:26,454 INFO    MainThread:41656 [wandb_run.py:_restore():2326] restore
2024-05-22 11:23:26,455 INFO    MainThread:41656 [wandb_run.py:_restore():2332] restore done
2024-05-22 11:23:31,196 INFO    MainThread:41656 [wandb_run.py:_footer_history_summary_info():3994] rendering history
2024-05-22 11:23:31,196 INFO    MainThread:41656 [wandb_run.py:_footer_history_summary_info():4026] rendering summary
2024-05-22 11:23:31,211 INFO    MainThread:41656 [wandb_run.py:_footer_sync_info():3953] logging synced files
